src/main/java/frc/robot/subsystems/constants/PivotConstants.java

```diff
@@ -72,48 +68,17 @@ public class PivotConstants {
               .s(kS)
               .v(kV)
               .a(kA)
-              // Motion Magic
-              .mmCruiseVelo(1),
-          80,
-          1600);
+              .maxTrapezoidalVelocity(kTrapezoidalMaxVelocity)
+              .maxTrapezoidalAcceleration(kTrapezoidalMaxAcceleration)
+              .maxTrapezoidalJerk(kTrapezoidalMaxJerk)
+              .expo_kV(kExponential_kV)
+              .expo_kA(kExponential_kA));
```

Comment:
it looks like this keeps MotionMagicExpo configs in place, so is the plan to keep working with expo? just wanted to ask since we briefly talked about simplifying things with just trapezoidal profiles (same question for ShoulderConstants.java)

Reply:
after looking at it, they're both complicated and require tuning, so i figure we can pick whichever we choose.  if we choose to go with trapezoidal, then these gains will just be ignored.